daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/TypeOrdering.scala

```diff
@@ -66,25 +66,26 @@ object TypeOrdering extends Ordering[Type] {
   }
 
   private[this] val builtinTypeIdx =
+    // must be in the same order as declared in the archive protobuf
```

Comment:
```suggestion
    // must be in the same order as declared in the archive protobuf
    // no need to be overly careful though since we have a test enforcing this
```